src/lnd-actions/gen-seed.js

```diff
@@ -0,0 +1,20 @@
+const { deadline } = require('../grpc-utils')
```

Comment:
This file is actually added in #106, not sure why it is still showing in the diff